linux/LinuxProcess.h

```diff
@@ -120,8 +120,8 @@ typedef struct LinuxProcess_ {
    double io_rate_write_bps;
    #endif
    #ifdef HAVE_OPENVZ
-   unsigned int ctid;
-   unsigned int vpid;
+   char* ctid;
```

Comment:
This new dynamic allocation needs to be cleaned up in
https://github.com/htop-dev/htop/blob/241e4b3dbf8ebee4a12a337dec725b3547c242e9/linux/LinuxProcess.c#L145-L153